docs/v3/status-socket.rst

```diff
@@ -0,0 +1,53 @@
+Status Socket
+=============
+
+Basic information about running OnionBalance can be obtained by querying
+so called status socket.
+
+Status socket is a Unix socket file created by OnionBalance. It is
+automatically closed by OnionBalance after reading it to the end.
+
+Example:
+
+.. code-block::
+
+    socat - unix-connect:/var/run/onionbalance/control | json_pp -json_opt pretty
+    {
+       "services" : [
+          {
+             "instances" : [
+                {
+                   "introModified" : "2020-06-16 19:59:28",
+                   "ipsNum" : 3,
+                   "onionAddress" : "vkmiy6biqcyphtx5exswxl5sjus2vn2b6pzir7lz5akudhwbqk5muead.onion"
+                }
+             ],
+             "onionAddress" : "bvy46sg2b5dokczabwv2pabqlrps3lppweyrebhat6gjieo2avojdvad.onion.onion",
+             "publishAttemptFirst" : "2020-06-16 20:00:12",
+             "publishAttemptSecond" : "2020-06-16 20:00:12"
+             "uploadedFirst" : "2020-06-16 20:00:12",
+             "uploadedSecond" : "2020-06-16 20:00:12"
+          }
+       ]
+    }
+
+The overall format of the status socket output is clear from the above
+example. Note that `ipsNum` and `introModified` for an instance is
+optional, `uploaded*` and `publishAttempt*` for a service may be `null`.
+
+Meaning of non-self-explanatory fields:
+
+* `introModified` is the intro set last modified timestamp.
+* `ipsNum` is the intro set size for a descriptor.
+* `publishAttemptFirst` and `publishAttemptSecond` are the last
+  publish attempt timestamps for first and second descriptors.
+* `uploadedFirst` and `uploadedSecond` are the last uploaded timestamps
+  for first and second descriptors.
+
+Configuration
+-------------
+
+Status socket filesystem location can be configured either by
+`status-socket-location` in the YAML config file
```

Comment:
Can you post an onionbalance config file that includes this option to see how it looks like in terms of operator UX?

Reply:
Added to docs.